tests/main/views/test_buyers.py

```diff
@@ -3723,13 +3723,9 @@ def test_that_no_option_chosen_triggers_error(self):
     def test_that_no_option_chosen_does_not_trigger_update(self):
         res = self.client.post(self.url.format(brief_id=123))
 
-        document = html.fromstring(res.get_data(as_text=True))
-        validation_message = document.xpath('//span[@class="validation-message"]')[0].text_content()
```

Comment:
Feels like there was some intent to check the content of the message, rather than deleting the unused thing it might be better to add an assertion here? (This can also count as a Flake8 fix I think - as it would no longer be an unused thing.)

Reply:
Ah looks like it was a copy paste error from the test above. That's checking the same content.